core/controllers/contributor_dashboard_test.py

```diff
@@ -201,8 +201,6 @@ def test_get_skill_opportunity_data(self):
             response['opportunities'], [
                 self.expected_skill_opportunity_dict_0,
                 self.expected_skill_opportunity_dict_1])
-
-        self.assertFalse(response['more'])
```

Comment:
Why are we dropping this and why are the rest of the changes in this file being made?

Reply:
`more` is no longer so realiable https://cloud.google.com/appengine/docs/standard/python/ndb/queryclass#Query_fetch_page

we can only check that it is `True` in cases when we know that there are more data. When we know that there are no more data it might be both `True` and `False`. If it is `False` then cool we don't need to make the next query, if it is `True` we need to make the next query but it might also be empty.

Reply:
Hmm, okay, it might be related to how the emulator works then, maybe the previous version made sure to return the correct value when emulating (since the test were working previously), now if I introduce these tests here they will start to flake (tested this before).


Not sure which signature to change, since the behaviour stays the same, we just cannot test one of the values.

Reply:
Done, I have decided to go with the double fetch solution for now, since after a quick evaluation of the FE code I think modifying backend is quicker.